manifest.json

```diff
@@ -194,7 +194,7 @@
               "dockerfile": "2.0/jessie/sdk",
               "os": "linux",
               "tags": {
-                "2.0.3-jessie": {},
```

Comment:
FYI - I never rebuilt this tag - I only fixed the manifest tags.

Reply:
Yeah, I'm going to rebuild d531fcfa86ce9ea27aa538774410176d9db6eb4a on CI.